doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst

```diff
@@ -6,9 +6,61 @@ New features, bug fixes, and improvements are regularly made to the SageMaker
 distributed model parallel library.
 
 
-SageMaker Distributed Model Parallel 1.11.0 Release Notes
+SageMaker Distributed Model Parallel 1.13.0 Release Notes
 =========================================================
 
+*Date: Dec. 15. 2022*
+
+**New Features**
+
+* Sharded data parallelism now supports a new backend for collectives, SMDDP. For supported scenarios
```

Comment:
nit:
"Sharded data parallelism now supports a new, more optimized process group backend option, "smdp," for collective calls. Under supported training configurations, "smddp" backend is enabled by default.

Reply:
I don't see that "smdp" is an option exposed externally, neither "smddp"

Reply:
Makes sense, from users' perspective they only see "nccl" or "auto". How about

"Sharded data parallelism is now accelerated by SMDDP Collectives. Under supported training configurations, "smddp" is automatically selected as the process group backend.

Reply:
incorporating using Arjun's suggestion